snowpack/src/build/build-import-proxy.ts

```diff
@@ -66,6 +67,20 @@ export function wrapHtmlResponse({
   // replace %PUBLIC_URL% in HTML files (along with surrounding slashes, if any)
   code = code.replace(/\/?%PUBLIC_URL%\/?/g, config.buildOptions.baseUrl);
```

Comment:
We want to support `%MODE%` as well, so that we can say that we support all env values in HTML.

Reply:
addressed in 5c35bd31